persistence/javadsl/src/main/java/com/lightbend/lagom/javadsl/persistence/DefaultPersistentEntityErrorHandler.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.persistence;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+public class DefaultPersistentEntityErrorHandler implements PersistentEntityErrorHandler {
+
+    static <Reply> CompletionStage<Reply> asFailedReply(Throwable failure){
+        CompletableFuture<Reply> failed = new CompletableFuture<>();
+        failed.completeExceptionally(failure);
+        return failed;
+    }
+
+
+    @Override
+    public <Reply, Cmd extends Object & PersistentEntity.ReplyType<Reply>> CompletionStage<Reply> handleAskFailure(Throwable failure, Cmd command) {
```

Comment:
Let me add more context. In `PersistentEntityRef` the left-most type bound is `Object` to maintain backwards compatibility (see https://github.com/lagom/lagom/commit/7651d07373fcfde736e50b15553ec2861e7f37e5) but since this is a new API we could drop it.

OTOH, considering where this method will be used, we can keep the left-most type bound as `Object` since it doesn't harm either. IDK...